superset/config.py

```diff
@@ -622,29 +624,31 @@ class CeleryConfig(object):
 # SQLALCHEMY_DATABASE_URI by default if set to `None`
 SQLALCHEMY_EXAMPLES_URI = None
 
-try:
-    if CONFIG_PATH_ENV_VAR in os.environ:
-        # Explicitly import config module that is not in pythonpath; useful
-        # for case where app is being executed via pex.
-        print(
-            "Loaded your LOCAL configuration at [{}]".format(
-                os.environ[CONFIG_PATH_ENV_VAR]
-            )
-        )
+if CONFIG_PATH_ENV_VAR in os.environ:
+    # Explicitly import config module that is not necessarily in pythonpath; useful
+    # for case where app is being executed via pex.
+    try:
+        cfg_path = os.environ[CONFIG_PATH_ENV_VAR]
         module = sys.modules[__name__]
-        override_conf = imp.load_source(
-            "superset_config", os.environ[CONFIG_PATH_ENV_VAR]
-        )
+        override_conf = imp.load_source("superset_config", cfg_path)
         for key in dir(override_conf):
             if key.isupper():
                 setattr(module, key, getattr(override_conf, key))
 
-    else:
+        print("Loaded your LOCAL configuration at [{}]".format(cfg_path))
+    except Exception:
+        logging.exception(
+            "Failed to import config for {}={}".format(CONFIG_PATH_ENV_VAR, cfg_path)
+        )
+        raise
+elif importlib.util.find_spec("superset_config"):
+    try:
         from superset_config import *  # noqa
         import superset_config
 
         print(
             "Loaded your LOCAL configuration at [{}]".format(superset_config.__file__)
         )
-except ImportError:
-    pass
+    except Exception:
+        logging.exception("Failed to import local superset_confg")
+        raise
```

Comment:
I'm a bit worried that we're changing the current behavior here, but IMHO if `superset_config.py` was found but not imported I think it's better to raise an exception.

So I think it's fine. :)

Reply:
Yeah, the thought crossed my mind, but I had a hard time contriving a reason where the current behavior, in the condition in which it is relevant, would be preferable to failing with a clear, actionable error message.

Reply:
Oh yes can confirm that before we just were trying not to raise if not found. But if found the module and failed the import we totally should raise!